app/src/main/res/values/dimens.xml

```diff
@@ -1,10 +1,25 @@
 <resources>
-    <!-- Default screen margins, per the Android Design guidelines. -->
-    <dimen name="activity_horizontal_margin">16dp</dimen>
-    <dimen name="activity_vertical_margin">16dp</dimen>
-    <dimen name="fab_margin">16dp</dimen>
-    <dimen name="app_bar_height">200dp</dimen>
-    <dimen name="item_width">200dp</dimen>
-    <dimen name="text_margin">16dp</dimen>
-    <dimen name="spacing_medium">16dp</dimen>
+
+    <dimen name="margin_extra_small">4dp</dimen>
```

Comment:
We can easily create a different spacing for that. I'm intentionally saying so they can be easily replaced and duplication is really bad. If we'll need to create new spacing, we'll do that.

The problem this way is that padding and margins are used interchangeably most of the time and they don't make a difference in most screens. If I quickly decide to change one dimen, it should be so across everything because material guidelines tell to keep constant spacings